samples/highcharts/data/livedata-spreadsheet/demo.js

```diff
@@ -0,0 +1,19 @@
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'spline',
+        height: 800
+    },
+    title: {
+        text: 'Live Data (Google Spreadsheets)'
+    },
+
+    subtitle: {
+        text: 'Data input from a remote spreadsheet w/polling'
+    },
+
+    data: {
+        googleSpreadsheetKey: '1zulHoZgTE8tFDfZ1I-Ngwd-czpuMOZrFnKoBPy87abg',
```

Comment:
This demo doesn't change over time - is it working?

Reply:
I couldn't find a way to make API fetches update the sheet (the triggers seem to only work when loading the sheet in sheets.google.com). I have however changed the sheet to use google finance, which should auto-update daily at least.